docs/core/deploying/index.md

```diff
@@ -1,98 +1,169 @@
+[Build your first .NET Core app using Visual Studio](https://docs.microsoft.com/dotnet/core/tutorials/with-visual-studio)
+ 
+[Build your first .NET Core app using the Command Line Interface (CLI)](../tutorials/using-with-xplat-cli.md)
 
-FDD and SCD deployments use separate host executables, so you can sign a host executable for an SCD with your publisher signature.
 
-### Why deploy a self-contained deployment?
+Either choice will allow you to create various types of .NET Core applications including ASP.NET Core, WPF, and Windows Forms applications.
 
-Deploying a Self-contained deployment has two major advantages:
 
-- You have sole control of the version of .NET Core that is deployed with your app. .NET Core can be serviced only by you.
+<br/>
 
-- You can be assured that the target system can run your .NET Core app, since you're providing the version of .NET Core that it will run on.
+### Step 2: Publish your app
 
-It also has a number of disadvantages:
+You can publish your app in one of two configurations - as a [Framework Dependent Deployment (FDD)](publish.md#Framework-Dependent-Deployment-FDD) or as a [Self-Contained Deployment (SCD)](publish.md#Self-contained-deployment-SCD).
 
-- Because .NET Core is included in your deployment package, you must select the target platforms for which you build deployment packages in advance.
+For both of the above configurations, you can publish the app either as a set of loose assemblies for either `xcopy` or package up for deployment, or alternatively as a single file executable.
 
-- The size of your deployment package is relatively large, since you have to include .NET Core as well as your app and its third-party dependencies.
+More information about each of these configurations can be found in the document on how to [Publish your app](publish.md).
 
-  Starting with .NET Core 2.0, you can reduce the size of your deployment on Linux systems by approximately 28 MB by using .NET Core [*globalization invariant mode*](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/globalization-invariant-mode.md). Ordinarily, .NET Core on Linux relies on the [ICU libraries](http://icu-project.org) for globalization support. In invariant mode, the libraries are not included with your deployment, and all cultures behave like the [invariant culture](xref:System.Globalization.CultureInfo.InvariantCulture?displayProperty=nameWithType).
 
-- Deploying numerous self-contained .NET Core apps to a system can consume significant amounts of disk space, since each app duplicates .NET Core files.
+<br/>
 
-## Framework-dependent executables (FDE)
+### Step 3: Package your app for deployment
```

Comment:
As I mentioned in the packaging guide, this section is really focused on Windows Desktop applications.  If there is not value in making this more broad, this section should set the expectations up front that it is scoped to Windows Desktop applications.

Reply:
Ack